src/js/MenuItem.js

```diff
@@ -0,0 +1,164 @@
+//MenuItem.js
+var Glyphicon = require('react-bootstrap/lib/Glyphicon.js');
+var Well = require('react-bootstrap/lib/Well.js');
+
+
+var MenuItem = React.createClass({
+  getInitialState: function(){
+      return {
+        flagged: false,
+        wrong: false,
+        needSwitch: false,
+        isCorrect: false,
+        displayWrong: true,
+        displayNeedSwitch: true,
+        displayIsCorrect: true
+      };
+  },
+
+  toggleFlag: function(e) { // this toggles the text as flagged or not flagged
```

Comment:
To comment a function, do it right outside the function itself. Line 18 in this case.